classlib/src/main/java/org/teavm/classlib/java/util/TCollections.java

```diff
@@ -460,6 +454,17 @@ public static int lastIndexOfSubList(TList<?> source, TList<?> target) {
         return -1;
     }
 
+    public static <T> TList<T> unmodifiableList(final TList<? extends T> list) {
```

Comment:
What was the reason to move this method? Such changes complicate review

Reply:
Just grouped together all umodifiable... implementations.